tools/device-listing/device-listing.cs

```diff
@@ -46,6 +46,7 @@ class Program
             "joystick",
             "color",
             "led",
+            "spi",
```

Comment:
note that this is not for devices using SPI, rather for stuff providing SPI (not sure if this category is useful for anything else than software SPI implementation but since it's not in System.Device.Gpio probably better to put it there for better visibility)

Reply:
It might be useful as well for spi bus expanders that work over i2c. I believe that is a thing although I don't have an example of such devices.

Reply:
we'll figure it out over time :smile: